.vs/BIPES/config/applicationhost.config

```diff
@@ -0,0 +1,1016 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please[ git ignore](https://git-scm.com/docs/gitignore) all .vs folder in the git ignore file like this:

```
.vs/
```

Reply:
Please reject changes t'you disagree or reject the Pull Request at all.

Cloud we proceed this way? I will make changes and re-resend it. I prefer to not set-up a new one repository. Do you have any suggestions that will be a better choice to rebase without need to fork another time?

Reply:
Well, you can undo commits  [(1)](https://stackoverflow.com/questions/448919/how-can-i-remove-a-commit-on-github) [(2)](http://www-cs-students.stanford.edu/%7Eblynn/gitmagic/ch05.html#_8230_and_then_some) but it's a little annoying.

Since it's actually just 10 files, I would destroy ``pete/BIPES`` on github and fork again, then in the terminal:
```
cd /var/www/html
cp BIPES BIPES_bck
rm -rf  BIPES
git clone git@github.com:pete-br/BIPES.git
```

and with a visual merge tool like [Gnome Meld](https://meldmerge.org/) [(2)](https://youtu.be/HcXixLB5x5s?t=576) [(3)](https://www.youtube.com/watch?v=TgrhAnTN-sE) apply changes from ``BIPES_bck`` into ``BIPES``.
Just **remember to backup** and setup git to [ignore ](https://www.atlassian.com/br/git/tutorials/saving-changes/gitignore)the vs code stuff right after cloning.

Also, [git stash](https://www.atlassian.com/br/git/tutorials/saving-changes/git-stash) and ``git stash pop`` are a great commands to save code and then reapply after a ``git pull``.

P.S. most links are in pt-br.

Reply:
Thakns Jorge